### DISCUSSION TITLE
combine lr scheduler with ReduceLROnPlateau

### DISCUSSION THREAD

**Discussion by maxmatical** [Lightning-AI/pytorch-lightning#10165] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
i'm trying to combine the `OneCycleLR` scheduler with `ReduceLROnPlateau`, is there a way to do this, for example

```
def configure_optimizers(self):
    optimizer = AdamW(...)
    scheduler1 = ReduceLROnPlateau(optimizer, ...)
    scheduler2 = OneCycleLR(optimizer, ...)
    return (
        {
            "optimizer": optimizer,
            "lr_scheduler": {
                "scheduler": scheduler1,
                "interval": "epoch",
                "monitor": "metric_to_track",
            },
        },
        {
            "optimizer": optimizer,
            "lr_scheduler": {
                "scheduler": scheduler2,
                "interval": "step"
            },
        }
    )

```

**Comment by rohitgr7** [comment]
passing it like:
```py
return [optimizer], [scheduler1_dict, scheduler2_dict] 
```
not working?
